ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Issue description
```

Comment:
Do we want to word this in the "What did you do? What did you expect to happen? What actually happened?" way?